### ISSUE TITLE
Importing the component causes propType error

### ISSUE THREAD

**Issue by el1f** [u2ix/react-background-slider#1]
When using the component as is the following error occours
```
Calling PropTypes validators directly is not supported by the `prop-types` package. Use PropTypes.checkPropTypes() to call them. Read more at http://fb.me/use-check-prop-types
```

**Comment by moronyoh** [comment]
I have the same issue

**Comment by u2ix** [comment]
Thanks for reporting the issue and submitting a PR!
Please verify that the problem is solved with the newly published package version `1.2.0`.
